org.cvfosammmm.Setzer.json

```diff
@@ -0,0 +1,98 @@
+{
+    "app-id": "org.cvfosammmm.Setzer",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.30",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.texlive"],
+    "command": "setzer",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=host",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--env=PATH=/app/bin:/usr/bin:/app/bin/x86_64-linux",
+        "--env=TEXMFCACHE=$XDG_CACHE_HOME",
+        "--env=LC_ALL=C",
+        "--env=TEXINPUTS=.:~/texmf/:",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "modules": [
+        {
+            "name": "texlive",
+            "buildsystem": "simple",
+            "build-commands": [
+                "/usr/lib/sdk/texlive/install.sh"
+            ]
+        },
+        {
+            "name": "gtksourceview",
+            "config-opts": ["--enable-gtk-doc=no"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/gtksourceview/3.24/gtksourceview-3.24.11.tar.xz",
+                    "sha256": "691b074a37b2a307f7f48edc5b8c7afa7301709be56378ccf9cc9735909077fd"
+                }
+            ]
+        },
+        {
+            "name": "openjpeg",
+            "buildsystem": "cmake-ninja",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/uclouvain/openjpeg/archive/v2.3.0.tar.gz",
+                    "sha256": "3dc787c1bb6023ba846c2a0d9b1f6e179f1cd255172bde9eb75b01f1e6c7d71a"
+                }
+            ]
+        },
+        {
+            "name": "poppler",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=release",
+                "-DENABLE_QT5:BOOL=true",
+                "-DCMAKE_INSTALL_LIBDIR:PATH=/app/lib"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://poppler.freedesktop.org/poppler-0.71.0.tar.xz",
+                    "sha256": "badbecd2dddf63352fd85ec08a9c2ed122fdadacf2a34fcb4cc227c4d01f2cf9"
+                }
+            ]
+        },
+        {
+            "name": "setzer",
+            "buildsystem": "simple",
+            "build-commands": [
```

Comment:
You probably need a proper build system here

Reply:
What do you mean by that? It's python, so there's no need to build anything.

Reply:
Ideally it should be done by some Makefile/setup.py/meson/etc, instead of having specific steps here. Just a general remark, not a blocker.

Reply:
Ok thanks for not blocking because of it, will review that in the future.